modules/openshift-storage-common-terms.adoc

```diff
@@ -0,0 +1,81 @@
+An iSCSI volume allows an existing iSCSI (SCSI over IP) volume to be mounted into your Pod.
+
+hostPath::
+A hostPath volume in an OpenShift Container Platform cluster mounts a file or directory from the host node’s filesystem into your pod.
+
+KMS key:: The Key Management Service (KMS) helps you achieve the required level of encryption of your data across different services. you can use the KMS key to encrypt, decrypt, and re-encrypt data.
+
+Local volumes:: A local volume represents a mounted local storage device such as a disk, partition or directory.
+
+NFS:: A Network File System (NFS) that allows remote hosts to mount file systems over a network and interact with those file systems as though they are mounted locally. This enables system administrators to consolidate resources onto centralized servers on the network.
+
+OpenShift Data Foundation::
+A provider of agnostic persistent storage for OpenShift Container Platform supporting file, block, and object storage, either in-house or in hybrid clouds
+
+Persistent storage::
+Pods and containers can require permanent storage for their operation. {product-title} uses the Kubernetes persistent volume (PV) framework to allow cluster administrators to provision persistent storage for a cluster. Developers can use PVC to request PV resources without having specific knowledge of the underlying storage infrastructure.
+
+Persistent volumes (PV):: {product-title} uses the Kubernetes persistent volume (PV) framework to allow cluster administrators to provision persistent storage for a cluster. Developers can use PVC to request PV resources without having specific knowledge of the underlying storage infrastructure.
+
+Persistent volume claims (PVCs):: You can use a PVC to mount a PersistentVolume into a Pod. You can access the storage without knowing the details of the cloud environment.
+
+Pod::
+One or more containers with shared resources, such as volume and IP addresses, running in your {product-title} cluster.
+A pod is the smallest compute unit defined, deployed, and managed.
+
+Reclaim policy::
+A policy that tells the cluster what to do with the volume after it is released. A volume’s reclaim policy can be `Retain`, `Recycle`, or `Delete`.
+
+Role-based access control (RBAC):: Role-based access control (RBAC) is a method of regulating access to computer or network resources based on the roles of individual users within your organization.
+
+Stateless applications:: A stateless application is an application program that does not save client data generated in one session for use in the next session with that client.
+
+Stateful applications:: A stateful application is an application program that saves data to persistent disk storage. A server, client, and applications can use a persistent disk storage. Stateful applications manage the deployment and scaling of a set of Pods, and provides guarantee about the ordering and uniqueness of these Pods.
```

Comment:
```suggestion
Stateful applications:: A stateful application is an application program that saves data to persistent disk storage. A server, client, and applications can use a persistent disk storage. Statefulset in {product-title} is used to manage the deployment and scaling of a set of Pods, and provides guarantee about the ordering and uniqueness of these Pods.
```

@xenolinux sorry, I need to add few words here compared to draft version.

Reply:
No worries :)

Addressed